crypto/rsa/rsa_lib.c

```diff
@@ -906,6 +906,56 @@ int ossl_rsa_get0_all_params(RSA *r, STACK_OF(BIGNUM_const) *primes,
     return 1;
 }
 
+#define safe_BN_num_bits(_k_)  (((_k_) == NULL) ? 0 : BN_num_bits((_k_)))
+int ossl_rsa_check_factors(RSA *r)
+{
+    int valid = 0;
+    int n, i, bits;
+    STACK_OF(BIGNUM_const) *factors = sk_BIGNUM_const_new_null();
+    STACK_OF(BIGNUM_const) *exps = sk_BIGNUM_const_new_null();
+    STACK_OF(BIGNUM_const) *coeffs = sk_BIGNUM_const_new_null();
+
+    if (factors == NULL || exps == NULL || coeffs == NULL)
+        goto done;
```

Comment:
Could you please log here that it's due to Memory allocation failure. This would help in debugging.

Reply:
@mattcaswell pointed out off-list, that such check happens inside of `OPENSSL_malloc()` already. So there is no reason to duplicate `OPENSSL_malloc()` work here.